README.md

```diff
@@ -1,20 +1,23 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+  # a click handler is a method that gets attached to a tag that you want to click.
```

Comment:
Yes. More accurately, a click handler listens for a click on a particular element, and says that a particular function will be run when someone clicks on that element.